{"Modified":"2022-08-27T03:19:00","Published":"2022-08-25T20:15:00","access":{},"assigner":"secalert@redhat.com","capec":[],"cvss":null,"cwe":"CWE-369","id":"CVE-2020-27802","impact":{},"last-modified":"2022-08-27T03:19:00","references":["https://github.com/upx/upx/issues/393"],"summary":"An floating point exception was discovered in the elf_lookup function in p_lx_elf.cpp in UPX 4.0.0 via a crafted Mach-O file.","vulnerable_configuration":[{"id":"cpe:2.3:a:upx_project:upx:4.0.0:*:*:*:*:*:*:*","title":"cpe:2.3:a:upx_project:upx:4.0.0:*:*:*:*:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:upx_project:upx:4.0.0:*:*:*:*:*:*:*"]}